src/net/relay.rs

```diff
@@ -689,7 +690,12 @@ impl Relayer {
     /// * list of confirmed microblock burn header hashes for newly-discovered microblock streams, so we can turn them into MicroblocksAvailable messages
     /// * list of unconfirmed microblocks that got pushed to us, as well as their relayers (so we can forward them)
     /// * list of neighbors that served us invalid data (so we can ban them)
-    pub fn process_new_blocks(network_result: &mut NetworkResult, burndb: &mut BurnDB, chainstate: &mut StacksChainState) -> Result<(Vec<BurnchainHeaderHash>, Vec<BurnchainHeaderHash>, Vec<(Vec<RelayData>, MicroblocksData)>, Vec<NeighborKey>), net_error> {
+    pub fn process_new_blocks(network_result: &mut NetworkResult, burndb: &mut BurnDB, chainstate: &mut StacksChainState)
```

Comment:
Can you edit the above comment to add `* list of chain tips and transaction receipts obtained by processing new blocks`?

Reply:
Also, can you open an issue to refactor this module so we have a single struct with all this data inside of it, instead of a long tuple?